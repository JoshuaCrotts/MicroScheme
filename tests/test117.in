(define (gcd a b)
  (if (= b 0)
      a
      (gcd b (% a b))))

(gcd 42 56)
(gcd 461952 116298)
(gcd 7966496 314080416)
(gcd 24826148 45296490)
(gcd 12 0)
(gcd 0 0)
(gcd 0 9)
