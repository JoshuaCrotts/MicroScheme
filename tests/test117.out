14
18
32
526
12
0
9
